src/js/test/index.test.js

```diff
@@ -43,11 +29,11 @@ describe('setupHeader', () => {
     it('adds `is-open` class to nav label', () => {
         // Arrange
         TestUtils.setBodyHtml(`
-            <input data-nav-enhance />
+            <button data-nav-button></button>
             <label data-nav-toggle>Menu</label>
         `);
         setupHeader();
-        const button = document.querySelector('button[data-nav-enhance]');
+        const button = document.querySelector('[data-nav-enhance]');
```

Comment:
Same here – Should be `data-nav-button` now? (would think this test should be failing as the button wouldn't be clicked?)